src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -388,14 +389,139 @@ uint32_t ContentAppPlatform::GetPincodeFromContentApp(uint16_t vendorId, uint16_
     return (uint32_t) strtol(pinString.c_str(), &eptr, 10);
 }
 
+constexpr EndpointId kTargetBindingClusterEndpointId = 0;
+constexpr EndpointId kLocalVideoPlayerEndpointId     = 1;
+constexpr EndpointId kLocalSpeakerEndpointId         = 2;
+// the fabric index can't be 0, but needs to be validated as 1
+constexpr FabricIndex kLocalFabricIndex   = 1;
+constexpr ClusterId kNoClusterIdSpecified = kInvalidClusterId;
+constexpr ClusterId kClusterIdOnOff       = 0x0006;
+constexpr ClusterId kClusterIdWakeOnLAN   = 0x0503;
+// constexpr ClusterId kClusterIdChannel             = 0x0504;
+// constexpr ClusterId kClusterIdTargetNavigator     = 0x0505;
+constexpr ClusterId kClusterIdMediaPlayback = 0x0506;
+// constexpr ClusterId kClusterIdMediaInput          = 0x0507;
+constexpr ClusterId kClusterIdLowPower        = 0x0508;
+constexpr ClusterId kClusterIdKeypadInput     = 0x0509;
+constexpr ClusterId kClusterIdContentLauncher = 0x050a;
+constexpr ClusterId kClusterIdAudioOutput     = 0x050b;
+// constexpr ClusterId kClusterIdApplicationLauncher = 0x050c;
+// constexpr ClusterId kClusterIdAccountLogin        = 0x050e;
+
+CHIP_ERROR ContentAppPlatform::ManageClientAccess(OperationalDeviceProxy * targetDeviceProxy, uint16_t targetVendorId,
+                                                  NodeId localNodeId, Controller::WriteResponseSuccessCallback successCb,
+                                                  Controller::WriteResponseFailureCallback failureCb)
+{
+    Access::AccessControl::Entry entry;
+    ReturnErrorOnFailure(GetAccessControl().PrepareEntry(entry));
+    entry.SetAuthMode(Access::AuthMode::kCase);
+    entry.SetFabricIndex(kLocalFabricIndex);
```

Comment:
should fabric index be passed in? How do we know 1 is always correct? The comments do not seem to help me: "the fabric index can't be 0, but needs to be validated as 1" just tells me that it cannot be 0 and I am not sure what "needs to be validated as 1" means.

Reply:
fixed to get actual fabric index